server/app/models/fusor/deployment.rb

```diff
@@ -12,8 +12,5 @@
 
 module Fusor
   class Deployment < ActiveRecord::Base
-    serialize :rhev_params, Hash
-    serialize :cfme_params, Hash
```

Comment:
Do we need any validations?  e.g.

```
    validates :name, :presence => true, :uniqueness => {:scope => :organization_id}
    validates :organization_id, :presence => true
    validates :lifecycle_environment_id, :presence => true
```

Not sure if this is something that would be handled by the foreman deployment feature in the future, but I suspect it won't account for lifecycle environment.